packages/zowe-explorer/src/extension.ts

```diff
@@ -43,6 +44,8 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
         uss: () => USSInit.initUSSProvider(context),
         job: () => JobInit.initJobsProvider(context),
     });
+
+    context.subscriptions.push(vscode.window.registerWebviewViewProvider("zowe-resources", TableViewProvider.getInstance()));
```

Comment:
Not sure how many more webviewView providers we may have in the future, but should we move this to an `initViewProviders` method or maybe under the `initializeProviders` method?

The only reason this crossed my mind was because this will be the only subscription directly in the activate method. Not saying that it's a bad thing, it's just that I'm afraid this might set a precedence for increasing the complexity of the `activate` /main method. 😋

Reply:
That makes sense to me - I'll move it into the `registerCommonCommands` function for now, as other webviews are defined there. If we choose to implement another `WebviewViewProvider` at some point, we can create an `initViewProviders` method to consolidate the initialization & registration of those providers.

Reply:
addressed in 861bb2c 😋